src/parser/jobs/pld/modules/Requiescat.tsx

```diff
@@ -79,6 +114,7 @@ export class Requiescat extends BuffWindow {
 
 	override initialise() {
 		super.initialise()
+		this.ignoreActions(EXCLUDED_ACTIONS.map(g => this.data.actions[g].id))
```

Comment:
Consider using this.trackOnlyActions instead, to limit the skills displayed in the rotation list to a (much shorter) whitelist instead of maintaining a large blacklist.

If that whitelist winds up being only Holy [Spirit|Circle] + Requiescat, you can also change the evaluator to be a native ExpectedGcdCountEvaluator and remove the custom RequiescatGcdsEvaluator.